canonical-web-and-design/jaas-dashboard
# Update site wide offline banner

## Done

Update site wide offline banner

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify changes requested in #694

## Details

 Fixes #694
